cider/tests/utils/CiderTestBase.cpp

```diff
@@ -105,6 +105,7 @@ void CiderJoinTestBase::assertJoinQuery(const std::string& sql,
 void CiderJoinTestBase::assertJoinQueryRowEqual(const std::string& sql,
                                                 const std::string& json_file,
                                                 const bool ignore_order) {
+  VLOG(4) << sql;
```

Comment:
Need revert?

Reply:
During generating EU, I see lots of log in omnisci's code, so I added one to debug. Do we prefer no log in test code?